src/Features/ListingPagePagination.php

```diff
@@ -28,7 +28,7 @@ protected static function name(): string {
 	 * @inheritDoc
 	 */
 	protected static function description(): string {
-		return __( 'Use the new paginated tag, category, author & post type listing pages.', 'planet4-master-theme-backend' );
+		return __( 'Use the new paginated tag, category, author & post and action type listing pages.', 'planet4-master-theme-backend' );
```

Comment:
Tiny detail, but I think the `&` here before `post` should be replaced by a comma 🙂